Gemfile

```diff
@@ -2,7 +2,7 @@
 # If you modify this file manually all comments and formatting will be lost.
 
 source "https://rubygems.org"
-gem "logstash-core", "2.0.0.dev", :path => "."
+gem "logstash-core", "2.0.0.snapshot1"
```

Comment:
@suyograo ^ this would work, IIRC bin/plugin update shouldn't update the core.

I've tested this version guard with:
```
source "https://rubygems.org"
gem "logstash-output-elasticsearch", "~> 2.0.0.beta"
```

`bundle install` command installed the latest version which is beta 2